knockapi/resources/users.py

```diff
@@ -349,3 +349,21 @@ def set_category_preferences(self, user_id, key, setting, options={}):
         params = setting if type(setting) is dict else {'subscribed': setting}
 
         return self.client.request('put', endpoint, payload=params)
+
+    ##
+    # Messages
+    ##
+
+    def get_messages(self, id, options=None):
```

Comment:
Is there a difference between passing in an empty dict like we are doing with the `set_category_preferences` func above vs defaulting to `None`? If not, i might just default to an empty dict to keep things consistent.

Reply:
@thomaswhyyou I copied this from https://github.com/knocklabs/knock-python/blob/464905cad760bf286bfa8c03cb241c8393c123cb/knockapi/resources/users.py#L81 which also supports pagination so I thought it made sense to follow that approach, what do you think?

Reply:
Ah i see. I'm pretty sure they don't make a difference, so given we already default to one or the other, probably fine to leave it :)